### ISSUE TITLE
Optimize solution

### ISSUE THREAD

**Issue by cathywu** [cathywu/TimeAlloc#38]
- [ ] See if there are variables that can be combined (e.g. chunking variables)
- [x] Perhaps the contiguity variables can be grouped by time (joined over tasks)
- [x] Early termination

Current problem:
```
Solve time 1293.3840870857239
```
```
Presolve 15189 (-20412) rows, 7502 (-9952) columns and 920980 (-1639883) elements
Statistics for presolved model
Original problem has 11088 integers (3696 of which binary)
Presolved problem has 7502 integers (1330 of which binary)
```

**Comment by cathywu** [comment]
Combined the contiguity variables into 2 (one for upper tri, one for lower tri). This should be a reduction of 528 --> 1 variables.

**NOTE**: The constraint/variable is now linearized, so the interpretation is less clear.

```
Solve time 324.3436210155487
```

Woohoo!

```
Presolve 2849 (-17972) rows, 1332 (-8732) columns and 14902 (-48141) elements
Statistics for presolved model
Original problem has 3698 integers (3696 of which binary)
Presolved problem has 1332 integers (1330 of which binary)
```

**Comment by cathywu** [comment]
Trying original contiguity constraints:
```
Presolve is modifying 2440 integer bounds and re-presolving
Presolve 15191 (-20413) rows, 8724 (-8733) columns and 928374 (-1643580) elements
Statistics for presolved model
Original problem has 11090 integers (3696 of which binary)
Presolved problem has 8724 integers (1330 of which binary)
```

**Comment by cathywu** [comment]
Resolved by #47, #45
